src/app/pages/registration/register-form/register-form.component.html

```diff
@@ -0,0 +1,150 @@
+                  <small class="text-danger" *ngIf="registerForm.controls.last_name.errors.required">Last Name is required.</small>
+              </div>
+                </div>
+            </div>
+
+            <div class="input-wrap">
+              <select class="role" formControlName="role"  placeholder=" Select a role">
+                <option value=""  value="null" disabled="true" [selected]="true">Select a role</option>
+                <option value="CA">Client Admin</option>
+                <option value="BY">Buyer</option>
+              </select>
+              <div *ngIf="!registerForm.get('role').valid && registerForm.get('role').touched" class="alert alert-danger">
+                Please select a role
+              </div>
+            
+            </div>
+
+            <div class="input-wrap">
+              
+                <div class="input-icons"> 
+                    <i class="fa fa-lock icon"> 
+                  </i> 
+              <input 
+              class="password"
+              formControlName="password"
+              placeholder="Create Password" 
+              type='password'/>
+
+              <div *ngIf="registerForm.controls.password.invalid && registerForm.controls.password.touched" class="alert alert-danger">
+                  <small class="text-danger" *ngIf="registerForm.controls.password.errors.required">password is required.</small>
+                  <small class="text-danger" *ngIf="registerForm.controls.password.errors.minlength">Password must contain an uppercase, number and a character</small>
+              </div>
+                </div>
+            </div>
+
+
+            <div class="input-wrap">
+                <div class="input-icons"> 
+                    <i class="fa fa-lock icon"> 
+                  </i> 
+              <input
+              class="password"
+              formControlName="confirmed_password"
+              placeholder="Confirm password" 
+              type='password' 
+                 />
+                 <div *ngIf="registerForm.controls.confirmed_password.invalid && registerForm.controls.confirmed_password.touched" class="alert alert-danger">
+                    <small class="text-danger" *ngIf="registerForm.controls.confirmed_password.errors.required">Confirm Password is required.</small>
+                    <div class ="text-danger" *ngIf="registerForm.controls.confirmed_password.errors.mustMatch">Passwords must match</div>
+                </div>
+                </div>
+            </div>
+
+            <button type='submit' [disabled]="!registerForm.valid">Create Account</button>
+        </form>
+        <p class="connect">Or Connect with</p>
+
+        <p class="social-media">
+          <span>
+            <img src="../../../../assets/img/images/ICON/Facebook.png" />
+          </span>
+          <span>
+            <img src="../../../../assets/img/images/ICON/LinkedIn.png" />
```

Comment:
use absolute imports here

Reply:
I need help with this @kelvinndmo... This is an import of an image.